bootstrap/kubeadm/internal/cloudinit/cloudinit_test.go

```diff
@@ -102,16 +105,13 @@ func TestNewInitControlPlaneCommands(t *testing.T) {
 	}
 
 	out, err := NewInitControlPlane(cpinput)
-	if err != nil {
-		t.Fatal(err)
-	}
+	g.Expect(err).NotTo(HaveOccurred())
+
 	expectedCommands := []string{
 		`"\"echo $(date) ': hello world!'\""`,
 		`"echo $(date) ': hello world!'"`,
 	}
 	for _, f := range expectedCommands {
-		if !bytes.Contains(out, []byte(f)) {
-			t.Errorf("%s\ndid not contain\n%s", out, f)
-		}
+		g.Expect(bytes.Contains(out, []byte(f))).To(BeTrue())
```

Comment:
```suggestion
		g.Expect(bytes).To(ConsistOf([]byte(f)))
```
Although, I do wonder if it might be better to use `BeEquivalentTo` here instead, because I suspect we care about order here.

Reply:
using `g.Expect(string(out)).To(ContainSubstring(f))`